src/f4enix/input/MCNPinput.py

```
@@ -601,6 +602,15 @@ def _to_dict(cards: list[parser.Card]) -> dict[str, parser.Card]:
                     comment = card.lines
                     flag_add = True
                     continue
+
+                # check if it is a column format
+                if PAT_COLUMN_FORMAT.match(card.lines[0]) is not None:
+                    # then this needs to be added to the previous card
+                    new_cards[previous_key].lines.extend(card.lines)
```

Ruff (0.15.2)

[error] Undefined name `previous_key`

(F821)